[2025-03-14T18:30:21.353+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-14T18:30:21.361+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: open_weather_etl.load scheduled__2025-03-14T18:25:00+00:00 [queued]>
[2025-03-14T18:30:21.495+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: open_weather_etl.load scheduled__2025-03-14T18:25:00+00:00 [queued]>
[2025-03-14T18:30:21.495+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-03-14T18:30:21.500+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): load> on 2025-03-14 18:25:00+00:00
[2025-03-14T18:30:21.504+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=314) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-03-14T18:30:21.504+0000] {standard_task_runner.py:72} INFO - Started process 315 to run task
[2025-03-14T18:30:21.505+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'open_weather_etl', 'load', 'scheduled__2025-03-14T18:25:00+00:00', '--job-id', '84', '--raw', '--subdir', 'DAGS_FOLDER/test_dag.py', '--cfg-path', '/tmp/tmpjj22agib']
[2025-03-14T18:30:21.506+0000] {standard_task_runner.py:105} INFO - Job 84: Subtask load
[2025-03-14T18:30:21.525+0000] {task_command.py:467} INFO - Running <TaskInstance: open_weather_etl.load scheduled__2025-03-14T18:25:00+00:00 [running]> on host a8e3300004fc
[2025-03-14T18:30:21.552+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='your_email@example.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='open_weather_etl' AIRFLOW_CTX_TASK_ID='load' AIRFLOW_CTX_EXECUTION_DATE='2025-03-14T18:25:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-14T18:25:00+00:00'
[2025-03-14T18:30:21.553+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-14T18:30:21.560+0000] {logging_mixin.py:190} INFO - starting loading process...
[2025-03-14T18:30:21.560+0000] {logging_mixin.py:190} INFO - Data Loaded into Postgres database
[2025-03-14T18:30:21.560+0000] {logging_mixin.py:190} INFO - data loading complete.
[2025-03-14T18:30:21.560+0000] {python.py:240} INFO - Done. Returned value was: Loading completed successfully!
[2025-03-14T18:30:21.567+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-14T18:30:21.567+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=open_weather_etl, task_id=load, run_id=scheduled__2025-03-14T18:25:00+00:00, execution_date=20250314T182500, start_date=20250314T183021, end_date=20250314T183021
[2025-03-14T18:30:21.604+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-14T18:30:21.610+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-14T18:30:21.611+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
